### DISCUSSION TITLE
doris数据增量备份时PROPERTIES中的type赋什么值呢

### DISCUSSION THREAD

**Discussion by leiyangcd** [apache/doris#12832] [A - General / Q&A]
备份语句:
BACKUP SNAPSHOT example_db.snapshot_label1
TO example_repo
ON (example_tbl)
PROPERTIES ("type" = "？？");
增量备份时上述语句的type赋什么值呢

**Comment by LemonLiTree** [comment]
目前支持最小分区（Partition）粒度的全量备份（增量备份有可能在未来版本支持）。如果需要对数据进行定期备份，首先需要在建表时，合理的规划表的分区及分桶，比如按时间进行分区。然后在之后的运行过程中，按照分区粒度进行定期的数据备份。
